src/main/resources/a/a_konfiguracja.xml

```diff
@@ -0,0 +1,20 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                           http://www.springframework.org/schema/beans/spring-beans.xsd
+                           http://www.springframework.org/schema/context
+                           http://www.springframework.org/schema/context/spring-context.xsd">
+    <context:component-scan base-package="ja.workshop.spring.części.konfiguracja.xml.a_wstrzykiwanie_zależności"/>
+    <!-- Definicja ziaren: -->
+
+
+    <!-- 3. Wstrzykiwanie ziarna fabryką ziaren -->
+    <!--<bean id="fabryczkaFasolekSilników" class="FabryczkaZiarenSilników"/>-->
```

Comment:
Zrób proszę GREP na tym kodzie i gdzie jest fasolka tam zrób ziarno. I miejmy to raz a dobrze z głowy. :-)